src/cloudWatchLogs/document/logStreamDocumentProvider.ts

```diff
@@ -35,6 +40,52 @@ export class LogStreamDocumentProvider implements vscode.TextDocumentContentProv
         }
         return content ?? ''
     }
+
+    async provideDefinition(
+        document: vscode.TextDocument,
+        position: vscode.Position,
+        token: vscode.CancellationToken
+    ): Promise<vscode.Definition | vscode.LocationLink[] | undefined> {
+        const activeUri = document.uri
+        const logGroupInfo = parseCloudWatchLogsUri(activeUri).logGroupInfo
+        if (logGroupInfo.streamName) {
+            // This means we have a stream file not a log search.
+            return
+        }
+        const curLine = document.lineAt(position.line)
+        try {
+            const streamIDMap = this.registry.getStreamIdMap(activeUri)
+            if (!streamIDMap || streamIDMap.size === 0) {
+                throw new Error(`cwl: No streamIDMap found for stream with uri ${activeUri}`)
```

Comment:
does the string form of `activeUri` potentially contain a big JSON blob? don't want that in an error message. can we get the URI without the querystring part?

Reply:
yeah, I can change it to `uri.path`.